src/index.ts

```diff
@@ -17,9 +17,53 @@
 
 import * as m from 'mithril';
 import App from './components/App';
+import './styles/global.scss';
+import BarlowRegularWoff from '../fonts/BarlowRegular.woff';
+import BarlowRegularWoff2 from '../fonts/BarlowRegular.woff2';
+import BarlowSemiBoldWoff from '../fonts/BarlowSemiBold.woff';
+import BarlowSemiBoldWoff2 from '../fonts/BarlowSemiBold.woff2';
 
-m.mount(document.body, App);
+// These should be identical to the @font-face rules in styles/_fonts.scss. Both
+// are included so there is a CSS fallback in case loading via JS fails for some
+// reason. Sadly, we can't just use the same family 'Barlow' for both of these
+// as we can in CSS (and toggle the font used just with the font-weight
+// property), as FontFace overwrites on loading the second one (unlike CSS,
+// which supports multiple @font-face rules for a single family).
+const barlowRegularFont = new FontFace(
+    'Barlow Regular',
+    `local('Barlow'), url('${BarlowRegularWoff2}') format('woff2'), url('${BarlowRegularWoff}') format('woff')`,
+    { style: 'normal', weight: '400' },
+) as FontFace;
+const barlowSemiBoldFont = new FontFace(
+    'Barlow SemiBold',
+    `local('Barlow'), url('${BarlowSemiBoldWoff2}') format('woff2'), url('${BarlowSemiBoldWoff}') format('woff')`,
+    { style: 'normal', weight: '600' },
+) as FontFace;
 
-// Disable browser pull to refresh so dragging to scroll m3 content doesn't
-// trigger a refresh (thereby turfing any unsaved content)
-document.getElementsByTagName('body')[0].style.overscrollBehavior = 'contain';
+/**
+ * Load the application.
+ */
+function loadApp() {
+    m.mount(document.body, App);
+
+    // Disable browser pull to refresh so dragging to scroll m3 content doesn't
+    // trigger a refresh (thereby turfing any unsaved content)
+    document.getElementsByTagName('body')[0].style.overscrollBehavior = 'contain';
+}
+
+// Must load fonts before mounting loading the application, otherwise initial
+// canvas render won't use the bundled custom webfonts (as it doesn't trigger
+// font loading via CSS like regular DOM elements). Always load the application
+// despite any errors loading the bundled fonts.
+barlowRegularFont.load().then((fontRegular: FontFace) => {
```

Comment:
It looks like there's no need for the type declaration here (`: FontFace`) as TS appears to have figured it out based on the type for `FontFace.load()`. Or are you think that specifying types in this situation are good for readability?